### ISSUE TITLE
Improve documentation structure

### ISSUE THREAD

**Issue by nikochiko** [Cloud-CV/EvalAI#2578]
#### The current picture:
The structure for the readthedocs documentation for EvalAI (https://evalai.readthedocs.io/en/latest/) is a bit hard to follow, The titles of different section as seen in the panel on the left look like random topics where two unrelated or two similarly titled topics are placed next to each other.

#### The improvement:
As the documentation covers a wide variety of topics, we should organize the topics into further sections with friendly names such as: **What is EvalAI?**, **How do I use it?** and **How to contribute?**. This is just one way, we can be more creative here.

For a visual idea of the sections, here is a snip from [Scrapy's documentation](https://scrapy.readthedocs.io/en/latest/), the sections and topics are excellently titled, which makes finding the required information super-easy:
![scrapy-rtd-left-panel](https://user-images.githubusercontent.com/37668193/71767273-871d4700-2f30-11ea-836d-039027248095.PNG)

**Comment by Suryansh5545** [comment]
I have recently work on a .rst files and it is quite hard in it to organize it to look good .

**Comment by Suryansh5545** [comment]
But I will try to work on it

**Comment by Suryansh5545** [comment]
@nikochiko Can i work on it?

**Comment by nikochiko** [comment]
@Suryansh5545 Sure! But make sure you also talk to a mentor so that they verify that this change is needed and that your PR can be merged. I'll ping them in cc here.
cc: @RishabhJain2018 @Ram81 @vkartik97

**Comment by adrijshikhar** [comment]
is this issue still open?

**Comment by Suryansh5545** [comment]
@adrijshikhar the issue is open but as no maintainer have given any though about it might not be good to proceed

**Comment by ayushsachan1** [comment]
Hey! I want to work on it
